process_report/tests/unit/processors/test_new_pi_credit_processor.py

```diff
@@ -202,7 +193,7 @@ def test_one_new_pi(self):
         self._assert_result_invoice_and_old_pi_file(
             invoice_month,
             test_invoice,
-            self.test_old_pi_file.name,
+            test_old_pi_file,
```

Comment:
While passing in a `Path` object does technically work `[pandas.read_csv](https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html)` does accept Path-like objects, our invoice and processor classes [expect the filepaths to be a str](https://github.com/CCI-MOC/invoicing/blob/211076bf5e6405917255c5b20f3ce7d3c22db133/process_report/processors/new_pi_credit_processor.py#L28). Please update your PR to make sure the filepath str is passed to the processors instead.